src/main/resources/application.yaml

```diff
@@ -82,4 +82,10 @@ auth.idam.client.detailsUri: "/o/userinfo"
 auth.provider.service.client.baseUrl: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
 idam.s2s-auth.totp_secret: ${S2S_SECRET:AAAAAAAAAAAAAAAA}
 idam.s2s-auth.microservice: rd_user_profile_api
-idam.s2s-auth.url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
\ No newline at end of file
+idam.s2s-auth.url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
+
+# resent user invite interval in minutes
+resendInterval: 60
+
+# sync time interval in minutes
+syncInterval: 60
```

Comment:
how would this be synchronized with profile sync if there is a change in profile sync schedule?